loom/graphics/gfxTexture.cpp

```diff
@@ -277,6 +282,8 @@ void bitmapExtrudeRGBA_c(const void *srcMip, void *mip, int srcHeight, int srcWi
 
 TextureInfo *Texture::load(uint8_t *data, uint16_t width, uint16_t height, TextureID id)
 {
+	id &= TEXTURE_ID_MASK;
```

Comment:
That's a fair point. OK, I'm fine with this. Should we generally speaking have a "resolveTextureId()" method that would have an error if passed with bad version?